clap_mangen/src/render.rs

```diff
@@ -111,6 +111,19 @@ pub(crate) fn options(roff: &mut Roff, cmd: &clap::Command) {
             body.push(roman(help));
         }
 
+        let possibles = &opt.get_possible_values();
+        if !possibles.is_empty() {
```

Comment:
Should this respect `is_hide_possible_values_set`?

Reply:
Yes it should. Should now be reflected.